OpenRA.Game/Map/ActorInitializer.cs

```diff
@@ -35,16 +39,39 @@ public ActorInitializer(Actor actor, TypeDictionary dict)
 			Dict = dict;
 		}
 
-		public T Get<T>() where T : IActorInit { return Dict.Get<T>(); }
-		public U Get<T, U>() where T : IActorInit<U> { return Dict.Get<T>().Value(World); }
-		public bool Contains<T>() where T : IActorInit { return Dict.Contains<T>(); }
+		public T GetOrDefault<T>(TraitInfo info) where T : IActorInit
+		{
+			return Dict.GetOrDefault<T>();
```

Comment:
This will be replaced with a proper implementation in the next PR (and likewise in `ActorPreviewInitializer`), so the other methods redundantly calling this one is intentional.